src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1010,8 +1010,19 @@ private List<AssociativeNode> GetModifiedNodes(Subtree subtree, bool redefinitio
                     // It can then be handled normally regardless of its ForceExecution state
                     subtree.ForceExecution = false;
 
+                    BinaryExpressionNode prevBNE = null;
+                    if (node is BinaryExpressionNode bne1){
+                        var prevNode = st.AstNodes.OfType<BinaryExpressionNode>().Where(x => (x.LeftNode as IdentifierNode).Name == bne1.LeftNode.Name);
```

Comment:
I would stick with using the `Value` property for IdentifierNode as it's specific to that node type while Name applies to the base `Node `type. May not make a difference but just for clarity sake.

Reply:
okay, will do.